docs/KNOWN_ISSUES.md

```diff
@@ -49,3 +63,31 @@ If you *must* call `player.reset()` then you'll have to manually restart the `vi
         // Add your plugin configuration here
     });
     ```
+
+## Setting start position on iOS safari
+
+On iOS Safari, there is a compatibility issue with the `loadedmetadata` event when attempting to set the start position
```

Comment:
It's a bit more than a compatibility issue as described https://bugs.webkit.org/show_bug.cgi?id=261512. It would be great to add a link to the bug report.

Reply:
I've added the link and changed the word "compatibility" for "bug" for consistence sake,